### PR TITLE
 Added support to override metric_url in newrelic output plugin

### PR DESCRIPTION
**Pull request by hsinghkalsi**
### Required for all PRs:

- [X ] Updated associated README.md.
- [ X] Wrote appropriate unit tests.

Added support to override Metric URL endpoint

### CODE REVIEW

File: plugins/outputs/newrelic/README.md [hunk 1]
```diff
@@ -21,7 +21,12 @@ Telegraf minimum version: Telegraf 1.15.0
...
+
+  ## Metric URL override to enable geographic location endpoints.
+  # If not set use values from the standard 
+  # metric_url = "https://metric-api.newrelic.com/metric/v1"
+
 ```
 
 [Metrics API]: https://docs.newrelic.com/docs/data-ingest-apis/get-data-new-relic/metric-api/introduction-metric-api
-[Insights API Key]: https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#user-api-key
+[Insights API Key]:  https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#user-api-key
```
Comment (srebhan) [inline plugins/outputs/newrelic/README.md:32]: Please revert the additional space here.

File: plugins/outputs/newrelic/README.md [hunk 2]
```diff
@@ -21,7 +21,12 @@ Telegraf minimum version: Telegraf 1.15.0
   ## HTTP Proxy override. If unset use values from the standard
   ## proxy environment variables to determine proxy, if any.
   # http_proxy = "http://corporate.proxy:3128"
+
+  ## Metric URL override to enable geographic location endpoints.
+  # If not set use values from the standard 
+  # metric_url = "https://metric-api.newrelic.com/metric/v1"
+
```
Comment (srebhan) [inline plugins/outputs/newrelic/README.md:28]: Please remove this newline.

**Review by srebhan — COMMENTED** [review]
@hsinghkalsi thanks for your work. This looks almost perfect except two small formatting comments for the README. Can you please fix them, the rest looks really good to me.

**Comment by hsinghkalsi** [comment]
@srebhan  Thanks for the review. I have removed the newline and extra space.

**Review by srebhan — APPROVED** [review]
Perfect. Thank you for the quick response.

**Review by ssoroka — APPROVED** [review]

**Pull request merged**